package.json

```diff
@@ -211,6 +214,10 @@
         "server-dev:validation": "npm run server-dev -- --env.component validation",
         "server-dev:vertical-navbar": "npm run server-dev -- --env.component vertical-navbar"
     },
+    "lint-staged": {
+        "*.yml": "yamllint"
+    },
+    "pre-commit": "lint-staged",
```

Comment:
we already have `husky` for create git hooks.

https://github.com/typicode/husky#install